### ISSUE TITLE
[BUG] ERR_CONNECTION_REFUSED 2.0 [latest version: 1.21.0-ls173]

### ISSUE THREAD

**Issue by Krushx0** [linuxserver/docker-freshrss#54]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

After the merge and the image have been repulled I still cannot reach the website and I have errors in the Container logs. With the current version: 1.21.0-ls173

### Expected Behavior

I can reach the website and use it as intended without having errors in the container logs.

### Steps To Reproduce

In this environment: docker + portioner using stack on a debian LXC container on proxmox. (every other "linuxserver" docker image works in this environment)
using the following docker compose:
version: "2.1"
services:
freshrss:
image: lscr.io/linuxserver/freshrss:latest
container_name: freshrss
environment:
- PUID=1000
- PGID=1000
- TZ=Europe/Budapest
volumes:
- /srv/dockerapps/freshrss/config:/config
- /srv/dockerapps/freshrss/extensions:/config/www/freshrss/extensions
ports:
- 49999:80
restart: unless-stopped

Deploy the stack
(without the extension volume row still the same behavior. I use the same path structure login for my other linuxserver docker image there is no problem with the path on my side. I give the proper 1000 puid an pgid for the folders as I did before for the other containers that works.)
You should wait for a 3-5 minutes to get the same result.
Check the container logs.

### Environment

```markdown
- OS: Proxmox -> LXC container -> debian-11-standard_11.6-1_amd64.tar.zst -> docker -> Portainer latest 
- How docker service was installed: following this guide: https://docs.docker.com/engine/install/debian/
- How Portainer installed: following this guide: https://docs.portainer.io/start/install-ce/server/docker/linux
```

### CPU architecture

x86-64

### Docker creation

```bash
---
version: "2.1"
services:
  freshrss:
    image: lscr.io/linuxserver/freshrss:latest
    container_name: freshrss
    environment:
      - PUID=1000
      - PGID=1000
      - TZ=Europe/Budapest
    volumes:
      - /srv/dockerapps/freshrss/config:/config
      - /srv/dockerapps/freshrss/extensions:/config/www/freshrss/extensions
    ports:
      - 49999:80
    restart: unless-stopped
```

### Container logs

```bash
[migrations] started
[migrations] 01-nginx-site-confs-default: skipped
[migrations] 02-default-location: skipped
[migrations] done
───────────────────────────────────────
      ██╗     ███████╗██╗ ██████╗ 
      ██║     ██╔════╝██║██╔═══██╗
      ██║     ███████╗██║██║   ██║
      ██║     ╚════██║██║██║   ██║
      ███████╗███████║██║╚██████╔╝
      ╚══════╝╚══════╝╚═╝ ╚═════╝ 
   Brought to you by linuxserver.io
───────────────────────────────────────
To support LSIO projects visit:
https://www.linuxserver.io/donate/
───────────────────────────────────────
GID/UID
───────────────────────────────────────
User UID:    1000
User GID:    1000
───────────────────────────────────────
using keys found in /config/keys
mv: cannot remove '/app/www/data/fever': Invalid argument
mv: cannot remove '/app/www/data/cache': Invalid argument
mv: cannot remove '/app/www/data/tokens': Invalid argument
mv: cannot remove '/app/www/data/users/_': Invalid argument
mv: cannot remove '/app/www/data/extensions-data': Invalid argument
mv: cannot remove '/app/www/data/favicons': Invalid argument
mv: cannot remove '/app/www/data/PubSubHubbub/feeds': Invalid argument
mv: cannot remove '/app/www/data/PubSubHubbub/keys': Invalid argument
mv: cannot remove '/app/www/extensions': Invalid argument
[custom-init] No custom files found, skipping...
[ls.io-init] done.
nginx: [emerg] no "ssl_certificate" is defined for the "listen ... ssl" directive in /config/nginx/site-confs/default.conf:3
nginx: [emerg] no "ssl_certificate" is defined for the "listen ... ssl" directive in /config/nginx/site-confs/default.conf:3
nginx: [emerg] no "ssl_certificate" is defined for the "listen ... ssl" directive in /config/nginx/site-confs/default.conf:3
nginx: [emerg] no "ssl_certificate" is defined for the "listen ... ssl" directive in /config/nginx/site-confs/default.conf:3
nginx: [emerg] no "ssl_certificate" is defined for the "listen ... ssl" directive in /config/nginx/site-confs/default.conf:3
nginx: [emerg] no "ssl_certificate" is defined for the "listen ... ssl" directive in /config/nginx/site-confs/default.conf:3
nginx: [emerg] no "ssl_certificate" is defined for the "listen ... ssl" directive in /config/nginx/site-confs/default.conf:3
nginx: [emerg] no "ssl_certificate" is defined for the "listen ... ssl" directive in /config/nginx/site-confs/default.conf:3
nginx: [emerg] no "ssl_certificate" is defined for the "listen ... ssl" directive in /config/nginx/site-confs/default.conf:3
nginx: [emerg] no "ssl_certificate" is defined for the "listen ... ssl" directive in /config/nginx/site-confs/default.conf:3
nginx: [emerg] no "ssl_certificate" is defined for the "listen ... ssl" directive in /config/nginx/site-confs/default.conf:3
nginx: [emerg] no "ssl_certificate" is defined for the "listen ... ssl" directive in /config/nginx/site-confs/default.conf:3
```

**Comment by j0nnymoe** [comment]
Could you try without a lxc container at all? They're known to cause problems with docker (even proxmox dev's don't recommend it).

**Comment by Krushx0** [comment]
Does it works for you? I mean a fresh install state with empty volumes attached? I running +10 linuxserver image only this one is acting up. If it works for you I could consider to try build other environment but until somebody confirm that is actually works I see no point to do that.

**Comment by j0nnymoe** [comment]
Our testing show's that it boots correctly: https://ci-tests.linuxserver.io/linuxserver/freshrss/latest/index.html

I don't personally use it myself.

**Comment by thespad** [comment]
A clean install works fine for me on both a standard VM host and in my test LXC environment (not proxmox).

**Comment by Krushx0** [comment]
I created a VM environment with debian, docker ,portainer but unfortunately it did not work I have the same problem. Just for a chance I tried the official image and it worked. I had hope that I could run linuxserver version that I always use and prefer but this time it failed me. I could simply not figure out what could cause the issue this time. For the time being I will stick to the official image.
